app/views/LicensorViewComponent.vue

```diff
@@ -79,14 +127,21 @@ div.licensor.container(v-else)
             th.border ID
             th.border Slug
             th.border Name
+            th.border minimumLicenseDays
             th.border No of license days assigned by client
             th.border No of users to whom licenses have been assigned
         tr(v-for = "client in clients")
             td.border {{client._id}}
             td.border {{client.slug}}
             td.border {{client.name}}
+            td.border {{client.minimumLicenseDays}}
             td.border {{client.licenses}}
             td.border {{client.users}}
+    label.border(v-if = "clients.length == 1" v-for = "client in clients")
```

Comment:
Do we support a single licensor user managing multiple API clients?  We need to if not.